src/NServiceBus.Core/Transports/Msmq/MsmqConstants.cs

```diff
@@ -0,0 +1,11 @@
+namespace NServiceBus.Transports.Msmq
+{
+    using System.Security.Principal;
+
+    static class MsmqConstants
```

Comment:
@danielmarbach ok, I thought about reusing those, but I ended up not to. I'll revert this.